encoding/video-artifacts.md

```diff
@@ -122,6 +139,7 @@ interlacing-related aliasing it is rarely recommendable.
 Ringing is something of a blanket term for edge artifacts, including
 mosquito noise, edge enhancement artifacts, overshoot, or actual
 ring-like ringing caused by the Gibbs phenomenon.
+![Mosquito Noise](images/mosquito1.png)\(Urban, 2017\)
```

Comment:
same

Reply:
Refer to [preivous answer](https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/pull/8#discussion_r254493425).